packages/react-topology/.storybook/preview-head.html

```diff
@@ -0,0 +1,10 @@
+<style>
```

Comment:
Do we really need a new storybook added just for topology? You can write MD files that get added to our https://patternfly-react.surge.sh docs like this:
``````
---
title: Name of page
section: topology
---
import { Whatever } from '@patternfly/react-topology';

```js title=Basic-topology
import { Whatever } from '@patternfly/react-topology';

BasicExample = () => (
  <Whatever />
)
```
``````

Reply:
storybook will be removed